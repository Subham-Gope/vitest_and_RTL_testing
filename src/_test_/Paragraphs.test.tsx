// import { describe, test, expect } from "vitest";
// import { render, screen } from "@testing-library/react";
// import Paragraphs from "../components/Paragraphs";

// describe("Paragraphs", () => {
//   test("renders two paragraphs", () => {
//    const  {container}=  render(<Paragraphs />);

//    expect(container).toMatchSnapshot();

//     const p1 = screen.getByText("One");
//     const p2 = screen.getByText("Two");

//     expect(p1).toBeInTheDocument();
//     expect(p2).toBeInTheDocument();
//   });
// });
