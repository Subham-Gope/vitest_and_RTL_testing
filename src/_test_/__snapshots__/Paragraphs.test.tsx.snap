// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`Paragraphs > renders two paragraphs 1`] = `
<div>
  <div>
    <p>
      One
    </p>
    <p>
      Two
    </p>
  </div>
</div>
`;
